vendor/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library(rocksdb_block_storage
```

Comment:
This is better placed in irohad/ametsuchi/impl directory, together with other implementations of block storage.

Reply:
noted